wss_tools/utils/recenter.py

```diff
@@ -0,0 +1,117 @@
+''' Module to recenter images '''
+import os
+from astropy.io import fits
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy import ndimage
+
+
+def rebin(arr, new_shape):
+    '''
+    Rebin 2D array arr to shape new_shape by averaging
+    :param arr:
+    :param new_shape:
+    :return: newarr - new array
+    '''
+    shape = (new_shape[0], arr.shape[0] // new_shape[0],
+             new_shape[1], arr.shape[1] // new_shape[1])
+    newarr = np.nanmedian(np.nanmedian(arr.reshape(shape), axis=-1), axis=1)
+
+    return newarr
+
+
+def recenter(images, outputdir, doplot=False):
+    '''
+    Recenter images to NRCA3 FP3 (464, 1412)
+    :param images:
+    :param outputdir:
+    :param doplot:
+    :return: List of output_images that have been recentered
+    '''
+    output_images = []
+    for im_fn in images:
+        # Open fits
+        hdul = fits.open(im_fn)
+        data = hdul[1].data
+
```

Comment:
We could test that the image is actually 2048 by 2048, but let's assume that will always be the case:
```suggestion
imsize = 2048
```